eval.sh

```diff
@@ -2,7 +2,6 @@
 mkdir -p ./eval/
 python benchmark.py --gold=./oie_corpus/all.oie --out=eval/Stanford.dat --stanford=./systems_output/stanford_output.txt
 python benchmark.py --gold=./oie_corpus/all.oie --out=eval/OLLIE.dat --ollie=./systems_output/ollie_output.txt
-python benchmark.py --gold=./oie_corpus/all.oie --out=eval/ReVerb.dat --reverb=./systems_output/reverb_output.txt
```

Comment:
Fixing Issue #12 breaks ReVerb. Removing it from eval.sh. It seems like the reverb_output.txt data is actually broken. The sentence IDs don't properly line up with the sentence ID's in the other ReVerb files.